src/main/resources/assets/schemas/FileStorageObject.yaml

```diff
@@ -0,0 +1,45 @@
+type: object
```

Comment:
right now, all fields are sent, but do we want all of these fields sent to the frontend?

Reply:
> right now, all fields are sent, but do we want all of these fields sent to the frontend?

My default answer is `yes` for all fields unless there is a compelling reason not to. There are cases where it doesn't make sense to expose certain model properties in certain cases (I think we do this for signing official names/ids on the user profile page), but those are usually the exception, not the rule.

Reply:
My only thought here is that maybe the GCS URI is a slight overcommunication to the user? I envision having an endpoint, e.g., `GET /api/fileStorageObject/{id}/content`, that would fetch GCS for the user. Thus, they would never need to know the GCS URI. 
I could also see that it would be considered a security concern to return it, but I am not sure if that's true.

Reply:
> My only thought here is that maybe the GCS URI is a slight overcommunication to the user?

Our GCS bucket is protected by service account access which minimizes the risk. Also, we currently return full path information for Consent files, for example:

Consents:
```
{
  "consentId": "eac1d4f9-78c9-4c88-9b10-9d692e171b5b",
  "lastElectionStatus": "Closed",
  "lastElectionArchived": false,
  "requiresManualReview": false,
  "dataUseLetter": "https://storage.googleapis.com/{bucket}/{filename}",
  ...
}
```

For DARs, we return the last part of that url which, if the user is well informed, could sort it out.
```
{
      ...
      "itDirector": "Test",
      "collaborationLetterLocation": "216f516b-c5f0-474f-8e67-c711979e2b26",
      "collaborationLetterName": "err.json",
      ...
}
```

However, I think you're correct, we should strive to minimize this kind of info. Let's pull that from the response.

Reply:
Good discussion.  I agree with the strategy of not exposing our internals (e.g. what cloud provider we may be using, etc) in responses.  Nice work!